apps/blog/src/posts/data-insights-on-PWAs/data-insights.md

```diff
@@ -0,0 +1,65 @@
+ * [Widgets](https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/widgets): Widgets are small applications that can be embedded within a larger application or operating system. They are designed to provide quick access to frequently used information or functionality. In the context of an app manifest, widgets are defined in the PWA manifest file using the widgets manifest member.
+
+## Insights from the data
+
+* It seems like  shortcuts are among the more popular features that developers like to use. Shortcuts are relatively easy to add, and are a quick way (or should a say, a shortcut), to add a feature that makes the PWA more app like. On the other hand, features such as share_target, file_handlers and widgets require more work to enable and are also only relevant for certain scenarios so it's understandable why they are not as popular. 
+
+* The number of PWAs with widgets might seem low, but it was actually a surprise to me as widgets are relatively a newer feature. Earlier this year, Facebook was one of the first big PWAs to add widgets to their manifest. Widgets are an easy way to increase engagement across different surfaces on an operating system. The uptick in widgets is a positive indicator of increasing awareness among developers. 
+
+* There are many apps with detected protocol handlers, but upon a closer look, a large number of them were in fact empty arrays. It’s likely that the developers are using manifest templates that already contain protocol handlers but are not actually using them. 
+
+ 
+## Parting thoughts
+
+From the data, it seems like developers are showing interest in adding app capabilities to improve the native-like experience of their PWA. As a developer, it's also important to understand and use these capabilities consciously only where it makes sense and avoid feature overload. Apps like [music player PWA](https://xamuzik.com/) are great examples of how to use app capabilities. This app has used many of the app capabilities, not only the ones listed above but others as well (handle_links, edge_side_panel, display_override). It's always great to find PWAs built by developers in the community that exemplify what a great app-like experience can be!
+
+While digging through the data, I stumbled upon this [music player PWA](https://xamuzik.com/) that had almost all the app capabilities. It’s available on multiple app stores including the Microsoft Store. The UX feels very app-like, and uses a lot of app capabilities that are relevant for this app and offer a rich native experience. Shout out to the developers! We love finding apps like these that exemplify what good PWAs are and how well PWAs can be integrated with the operating system. 
```

Comment:
You mention this app twice, is that intended?

Reply:
You can also end with asking folks to let us know what other data they's like to see in the future

Reply:
OOPS I thought I removed it good call out!

Reply:
> You can also end with asking folks to let us know what other data they's like to see in the future

Where can I ask them to reach out? Discord? Github?

Reply:
Discord makes more sense.